packages/network/test/unit/cors_spec.ts

```diff
@@ -211,6 +211,127 @@ describe('lib/cors', () => {
     })
   })
 
+  context('.urlOriginsMatch', () => {
+    beforeEach(function () {
+      this.isFalse = (url1, url2) => {
+        expect(cors.urlOriginsMatch(url1, url2)).to.be.false
+      }
+
+      this.isTrue = (url1, url2) => {
+        expect(cors.urlOriginsMatch(url1, url2)).to.be.true
+      }
```

Comment:
Maybe `assertOriginsDontMatch` and `assertOriginsMatch` instead? I feel like `isFalse/isTrue` should return whether the argument is false or true.

These could also be defined in the context's scope so there wouldn't be a need to use `this.` everywhere, e.g.:

```js
  context('.urlOriginsMatch', () => {
    const assertOriginsDontMatch = () => { /* ... */ }
    
    it('test', () => {
      assertOriginsDontMatch('https://foo.bar:443', this.url)
      assertOriginsDontMatch('https://foo.bar:80', this.url)
      // .. and so on ...
    })
  })
```

Reply:
I like that style a lot better. I went through the other tests in `lib/cors` and refactored them in 8404853381